guides/doc-Upgrading_and_Updating/topics/upgrading_satellite_server_parent.adoc

```diff
@@ -23,3 +29,7 @@ You cannot upgrade {ProjectServer} from a RHEL base system that is not operating
 
 To run {ProjectServer} on a {RHEL} base system operating in FIPS mode, you must install {Project} on a freshly provisioned RHEL base system operating in FIPS mode.
 For more information, see {InstallingProjectDocURL}preparing-environment-for-satellite-installation[Preparing your environment for installation] in _Installing {ProjectServer}_.
+
+ifdef::satellite[]
+include::migrating_pulp_content.adoc[leveloffset=+1]
+endif::[]
```

Comment:
Why this is only included for Satellite? I believe it is valid for katello too, so this should be:

    ifdef::katello,satellite[]

Reply:
fixed